scripts/update_codegen_tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Updates codegen tests' expected LLVM IR
+#
+
+set -eu
+
+# Change dir to project root
+cd "$(dirname "${BASH_SOURCE[0]}")"
+cd ..
+
+# Build docker image
+pushd docker
+docker build                  \
+  --network host              \
+  --build-arg LLVM_VERSION=8  \
```

Comment:
I think i based it on 7, but 8 is identical from what I've seen

Reply:
8 seemed to work OK. Could do 7 but maybe 8 will have a longer shelf life. Up to you

Reply:
8 seem to work fine so lets stick to it :)